### PR TITLE
[BugFix][MLA] Fix attn_mask bug for ring mla

### PR DESCRIPTION
**Pull request by whx-sjtu**
This PR fix a bug related to attention mask used in ring mla. Current ring mla has supported compressed mask, so we can directly use a 512 * 512 attention mask.

### CODE REVIEW

File: vllm_ascend/attention/mla_v1.py [hunk 1]
```diff
@@ -483,10 +483,16 @@ def __init__(
         self.enable_kv_nz = ascend_config.torchair_graph_config.enable_kv_nz
         self.chunked_prefill_for_mla = ascend_config.chunked_prefill_for_mla
 
-        self.prefill_mask = None
+        vllm_config = get_current_vllm_config()
+        self.prefill_mask = torch.triu(
+            torch.ones(512,
+                       512,
+                       device="npu",
```
Comment (MengqingCao) [inline vllm_ascend/attention/mla_v1.py]: will this always create mask on rank 0? is there any way to get current device?

Comment (whx-sjtu) [reply vllm_ascend/attention/mla_v1.py]: This is called in a process that has already set_device, in this situation 'npu' will be automatically set to corresponding rank in set_device

**Comment by wangxiyuan** [comment]
please fix the UT asap

**Comment by wangxiyuan** [comment]
the new commit is only for ut fix. it's good to go

File: vllm_ascend/attention/mla_v1.py [hunk 2]
```diff
@@ -483,10 +483,12 @@ def __init__(
         self.enable_kv_nz = ascend_config.torchair_graph_config.enable_kv_nz
         self.chunked_prefill_for_mla = ascend_config.chunked_prefill_for_mla
 
+        vllm_config = get_current_vllm_config()
+        self.ring_mla_mask_size = 512
         self.prefill_mask = None
 
         # Adapt torch air graph mode with spec decoding.
-        speculative_config = get_current_vllm_config().speculative_config
+        speculative_config = vllm_config.speculative_config
```
Comment (momo609) [inline vllm_ascend/attention/mla_v1.py:491]: does 091dev have the same bug?
